### ISSUE TITLE
Interface active power and apparent power limits

### ISSUE THREAD

**Issue by sylvlecl** [powsybl/pypowsybl#336]
* **Do you want to request a *feature* or report a *bug*?**

Feature.

* **What is the current behavior?**

Only current limits can be displayed.

* **What is the expected behavior?**

We should be able to display / modify all kinds of limits on branches: active power, apparent power, current.

The best option is probably to "replace" the current limits dataframe with a more generic operational_limits dataframe:

```python
network.get_operation_limits()
```
Which would give, for example, with different types of limits and different type of elements:

| element_id | element_type | side | type | name | value | acceptable_duration | is_fictitious |
| -- | -- | -- | -- | -- | -- | -- | -- |
| 'my_line' | LINE | ONE | CURRENT | '20min' | 1000 | 1200 | False |
| 'my_line' | LINE | ONE | CURRENT | '10min' | 1500 | 600 | False |
| 'my_line' | LINE | TWO | APPARENT_POWER | '20min' | 1000 | 1200 | False |
| 'my_line' | LINE | TWO | ACTIVE_POWER | '20min' | 1000 | 1200 | False |
| ... | ... | ... | ... | ... | ... | ... | ... |
| 'my_transfo_3' | T3T | ONE | CURRENT | '20min' | 1000 | 1200 | False |
| 'my_transfo_3' | T3T | TWO | CURRENT | '20min' | 1000 | 1200 | False |
| 'my_transfo_3' | T3T | THREE | CURRENT | '20min' | 1000 | 1200 | False |
| ... | ... | ... | ... | ... | ... | ... | ... |
| 'my_dangling_line' |  DL |  | CURRENT | '20min' | 1000 | 1200 | False |

* **What is the motivation / use case for changing the behavior?**

For example, matpower import creates apparent power limits, which are not accessible with pypowsybl.

**Comment by sylvlecl** [comment]
For the modification part, I propose to implement it as on the java side:
creating limits of a given type will actually replace the previous ones of that type.

```python
# will delete previous current limits at side 1
# and create a new one
network.create_operational_limits(element_id='my_line', element_type='LINE', side='ONE',
        type='CURRENT', name='20min', value=1000, acceptable_duration=1200)
```

Permanent limits could be defined by using `np.inf` value for acceptable_duration:
```python
network.create_operational_limits(element_id='my_line', element_type='LINE', side='ONE',
        type='CURRENT', value=1000, acceptable_duration=np.inf)
```
Implementation detail to be checked: how does np.inf translates to C and java sides?

In order to be able to delete all limits, we can provide an additional method:
```python
network.remove_operational_limits(element_id='my_line', element_type='LINE', side='ONE', type='CURRENT')
```

**Comment by EtienneLt** [comment]
solved with #341
